README.md

```diff
@@ -38,6 +38,11 @@ services like message brokers that exists _somewhere_ in a simple way:
     application configuration primitives in Kubernetes: Services, Secrets, and
     ConfigMaps.
 
+Video links:
```

Comment:
I just don't think they should be spread out with some video links up top and others down below.  Just want them all in one spot - no opinion on where in the doc.